src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheckTest.java

```diff
@@ -0,0 +1,64 @@
+package org.openstreetmap.atlas.checks.validation.linear.edges;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Tests for {@link MalformedRoundaboutCheck}
+ *
+ * @author savannahostrowski
+ */
+public class MalformedRoundaboutCheckTest
+{
+
+    private final MalformedRoundaboutCheck check = new MalformedRoundaboutCheck(
+            ConfigurationResolver.emptyConfiguration());
```

Comment:
This check is not stateless (processed features are flagged). Therefore, the order of processing could change the result. I'd suggest passing in a new instance of the check per unit test.